eq_cir_proxy_service/services/instrument/instrument_retrieval_service.py

```diff
@@ -49,19 +40,21 @@ async def retrieve_instrument(instrument_id: UUID) -> Instrument:
             },
         )
 
-    url = f"{cir_base_url}{cir_endpoint}"
-    try:
-        async with AsyncClient() as client:
-            response = await client.get(url, params={"guid": str(instrument_id)})
-    except RequestError as e:
-        logger.exception("Error occurred while retrieving instrument.", error=e)
-        raise HTTPException(
-            status_code=500,
-            detail={
-                "status": "error",
-                "message": "Error connecting to CIR service.",
-            },
-        ) from e
+    async with get_api_client(
+        url_env="CIR_API_BASE_URL",
+        iap_env="CIR_IAP_CLIENT_ID",
+    ) as cir_api:
```

Comment:
Shouldn't this be called `..client` for clarity? since `get_api_client` returns a client that later you use (calling client's get() method?

Reply:
Yes - have renamed here and in the converter service usage